code/ExamplePlayer.Ragdoll.cs

```diff
@@ -6,7 +6,10 @@
 
 partial class ExamplePlayer
 {
-    static EntityLimit RagdollLimit = new EntityLimit { MaxTotal = 20 };
+    //==============================================
+    //REDUCED ENTITY LIMIT TO 10 AS IT CAUSES LAG
+    //==============================================
```

Comment:
That's handy for me to know why you changed it but it doesn't belong in the code, you should make this a single line comment e.g. `// Can create lag, be cautious setting this too high`